test/parallel/test-https-simple.js

```diff
@@ -45,13 +45,13 @@ const serverCallback = common.mustCall(function(req, res) {
 });
 
 const invalid_options = [ 'foo', 42, true, [] ];
-invalid_options.forEach((option) => {
+for (const option in invalid_options) {
```

Comment:
`for ... in` will _not_ work here as it will iterate over keys, not values.

```suggestion
for (const option of invalid_options) {
```